test/opt/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 add_subdirectory(dominator_tree)
+add_subdirectory(loop_optimisations)
```

Comment:
We have been using the other spelling "optimization" in this project.

Reply:
Good point.